### ISSUE TITLE
cannot find resource

### ISSUE THREAD

**Issue by googya** [boazsegev/plezi#3]
tmp  $ plezi new conn
/Users/leslie/.rvm/gems/ruby-2.2.0/gems/plezi-0.7.0/bin/plezi:103:in `read': No such file or directory @ rb_sysopen - /Users/leslie/.rvm/gems/ruby-2.2.0/gems/plezi-0.7.0/resources/plezi_gray.png (Errno::ENOENT)
	from /Users/leslie/.rvm/gems/ruby-2.2.0/gems/plezi-0.7.0/bin/plezi:103:in `initialize'
	from /Users/leslie/.rvm/gems/ruby-2.2.0/gems/plezi-0.7.0/bin/plezi:255:in `new'
	from /Users/leslie/.rvm/gems/ruby-2.2.0/gems/plezi-0.7.0/bin/plezi:255:in `<top (required)>'
	from /Users/leslie/.rvm/gems/ruby-2.2.0/bin/plezi:23:in `load'
	from /Users/leslie/.rvm/gems/ruby-2.2.0/bin/plezi:23:in `<main>'
	from /Users/leslie/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `eval'
	from /Users/leslie/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `<main>'

**Comment by boazsegev** [comment]
Hi Googya,

Thank you for pointing this out. it was in issue caused when we changed the framework's name and forgot to update one of the file's name.

The issue was resolved in version 0.7.1... which will be released shortly.

I was waiting with releasing v. 0.7.1 because some changes might break older code and I was waiting for more changes to occur (and maybe to bump to a version 0.8.0, to make it clear that older code might break).

I am sorry for any inconvenience and I'm releasing the updated version as I write this.
